test/unit/core/Object3D.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @author simonThiele / https://github.com/simonThiele
+ */
+
+module( "Object3D" );
+
+test( "rotateX", function() {
+  var obj = new THREE.Object3D();
+
+  // get a reference object for comparing
+	var reference = obj.rotateOnAxis(new THREE.Vector3(1, 0, 0), 45).rotation;
+  obj.rotateX(45);
```

Comment:
`rotateX()` calls `rotateOnAxis()`. You are not actually testing anything here.

This appears to be an issue throughout this PR.

Also, Euler rotations are not unique. I do not think it is wise to compare them. Perhaps comparing the quaternions is best. But even so, the negation of a unit quaternion represents the same rotation.